packages/frontend/src/components/connect-wallet/index.tsx

```diff
@@ -68,28 +68,27 @@ export const ConnectWallet = () => {
     const chainId = chain?.id || Number.MAX_SAFE_INTEGER;
     // TODO: i18n
     const chainName = chain?.name || "Unknown";
+    console.log({ chain, chainId });
```

Comment:
`console.log` here.

Reply:
Removed in e9f10d91843cfa0700808797257dc6bd94447358